src/Home.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import {
+  View,
+  Text,
+  TextInput,
+  ScrollView,
+  TouchableOpacity,
+  KeyboardAvoidingView,
+  Platform,
+  SafeAreaView,
+} from 'react-native';
+import styles from './styles';
+
+import DaySelector from './components/DaySelector';
+import MealsComponent from './components/Meals';
+
+type DayOfWeek = {
+  short: string;
+  full: string;
+};
+
+const HomePage: React.FC = () => {
+  const [meals, setMeals] = useState<string[]>(['', '', '']);
```

Comment:
why not like this?
```suggestion
  const [meals, setMeals] = useState<string[]>([]);
```

Reply:
ok, now that I read more from the PR I can see you're counting on this to be always 3 strings / meals but it's not good practice as you're limiting this.  What if someone wants to add a brunch or tea time? you shouldn't be limiting in advance the amount of options

Reply:
ok, fixed that